examples/Huggingface_Transformers/Transformer_handler_generalized.py

```diff
@@ -142,27 +141,3 @@ def inference(self, inputs):
             logger.info("Model predicted: '%s'", prediction)
 
         return [prediction]
-
```

Comment:
Much of the `initialize` code seems to replicate what's in BaseHandler -- e.g. loading the model, pushing it to a device, etc. Could you call `super().initialize(ctx)` before adding in the rest?

As a side note, we should probably standardize a way to customize the handlers' behavior.